Dockerfile

```diff
@@ -66,6 +66,10 @@ RUN scripts/replace-placeholder.sh http://NEXT_PUBLIC_WEBAPP_URL_PLACEHOLDER ${N
 
 FROM node:18 as runner
 
+RUN apt-get update \
+    && apt-get install -y netcat-openbsd \
+    && apt-get clean \

```

Comment:
The clean should not be required anymore, as all major debian based images have apt hooks that perform a clean automatically. Only the "rm -rf" is required nowadays.